### DISCUSSION TITLE
problem with the style of the site

### DISCUSSION THREAD

**Discussion by roby444** [community/community#132074] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

(sorry for my bad english)i’m trying to publish a site with an index file and a style file, but when open the site there is the html of the index file but the style of the css file is not there, how can i solve it?

**Comment by catunderrain** [comment, marked as answer]
Have you import the style.css file to your html code? Like:
`<link rel="stylesheet" href="style.css">` and must to be in the same folder as the html one.
or inline style:
`<h1 style="color:blue;text-align:center;">This is a heading</h1>`
Hope you get well with your issue bro!

**Reply by roby444** [reply]
bro thanks you sooo much, i tried the second tip and it worked but could you explain me why it worked ? like why does “inline mean? sorry i’m a beginner

**Reply by catunderrain** [reply]
Bro, there are three ways to implement CSS styles in an HTML document: internal, external, and inline. You can look at this website for a clearer explanation: https://www.w3schools.com/css/css_howto.asp. We call it "inline" because we put the style configuration directly inside a tag. It's easy to remember which tag we styled, but for global styling, it takes much time to code and is hard to manage.
